bower.json

```diff
@@ -5,10 +5,16 @@
     "dist/rollbar.umd.js"
   ],
   "ignore": [
+    "dist/*.snippet*",
+    "dist/*.nojson*",
+    "dist/*.named-amd*",
```

Comment:
If you're using bower only to perform the download, then pointing your build to a file inside the resulting directory, then all you should need to do is instead switch to using the npm version, executing a build, and pointing to the named-amd output in dist/.

The typical use case for bower involves automatically consuming the "main" entry of a project's bower.json, which in this case differs from you guys consumption.  I would recommend switching to consuming rollbar.js from npm and changing your build appropriately.